certbot/tests/main_test.py

```diff
@@ -656,7 +670,7 @@ def test_certonly_abspath(self):
         chain = 'chain'
         fullchain = 'fullchain'
 
-        with mock.patch('certbot.main.obtain_cert') as mock_obtaincert:
+        with mock.patch('certbot.main.certonly') as mock_obtaincert:
```

Comment:
This should probably be `mock_certonly`, although I care less than I do about `afa` because this name is much better.